lib/screens/department_screen/department_screen.dart

```diff
@@ -0,0 +1,194 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+import 'package:tsec_app/screens/department_screen/widgets/about_section.dart';
+import 'package:tsec_app/screens/department_screen/widgets/faculty_details_section.dart';
+import 'package:tsec_app/utils/themes.dart';
+
+import 'widgets/department_screen_app_bar.dart';
+import 'widgets/drop_down_menu_item.dart';
+
+enum Item {
+  About,
+  FacultyDetails,
+  PlacementDetails,
+  ResultAnalysis,
+  Curriculum,
+  More,
+}
```

Comment:
Prefer lowerCamelCase [ref](https://dart.dev/tools/linter-rules#:~:text=In%20new%20code%2C%20use%20lowerCamelCase%20for%20constant%20variables%2C%20including%20enum%20values.)